### PR TITLE
[SPARK-31352] Add .asf.yaml to control Github settings

### PR DESCRIPTION
**Pull request by kaxil**
### What changes were proposed in this pull request?

Jira: https://issues.apache.org/jira/browse/SPARK-31352
We added this to Apache Airflow project ([PR](https://github.com/apache/airflow/pull/6689)) and I think it would be good to have the Spark's website at the top.

Also, this would allow Spark's PMC Members and committer to control common Github project settings by themselves without having to ask Apache INFRA team.

More info: https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories

We (Airflow PMC and Committers) used this file to enable Github issues and disable PR Merge button etc

### Why are the changes needed?

Explained above

### Does this PR introduce any user-facing change?

No

### How was this patch tested?

n/a

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
I think it seems fine to have. Mind filing a JIRA please? See also https://spark.apache.org/contributing.html

**Comment by kaxil** [comment]
> I think it seems fine to have. Mind filing a JIRA please? See also https://spark.apache.org/contributing.html

Updated

**Comment by HyukjinKwon** [comment]
ok to test

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+---
+github:
+  description: "Apache Spark - A unified analytics engine for large-scale data processing"
+  homepage: https://spark.apache.org/
+  labels:
+    - python
```
Comment (srowen) [inline .asf.yaml:22]: Sounds good to me. Later we could revisit these tags but this is fine as a copy of the current ones.

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**